collect_app/build.gradle

```diff
@@ -97,9 +97,8 @@ android {
             }
         }
 
-        // This uses the default debug keystore that the Android tooling creates https://stackoverflow.com/questions/16622528/android-studio-debug-keystore
         selfSignedRelease {
-            storeFile new File(System.getProperty("user.home") + '/.android/debug.keystore')
+            storeFile file('../debug.keystore')
```

Comment:
We were originally using the Android Studio `debug.keystore` for convenience, but this requires a full installation of Android Studio (rather than just having the Android SDK tools installed).